lib/posix/pthread.c

```diff
@@ -12,6 +12,20 @@
 #include <posix/pthread.h>
 #include <sys/slist.h>
 
+#ifndef ARCH_MIN_STACK_SIZE
+/* Pay no attention to the man behind the curtain! */
+#define ARCH_MIN_STACK_SIZE 256
```

Comment:
Is anyone aware of an existing lower bound that would be appropriate here?

Reply:
can you put a more descriptive comment on what this stack size means, and why we need to set a lower bound?

Reply:
I guess it's equivalent of the smallest useful stack frame, which could be zero, technically speaking, but if we don't have anything like this already, then I'll just remove places that are checked for it.